src/Compilers/CSharp/Test/Symbol/Symbols/PartialPropertiesTests.cs

```diff
@@ -4450,6 +4450,87 @@ partial class C
                 Diagnostic(ErrorCode.ERR_InvalidModifierForLanguageVersion, "this").WithArguments("partial", "12.0", "preview").WithLocation(7, 24));
         }
 
+        [Fact]
+        public void GetDeclaredSymbol_01()
+        {
+            var source = ("""
+                partial class C
+                {
+                    public partial int Prop { get; }
+                    public partial int Prop { get => 1; }
+                }
+                """.NormalizeLineEndings(), "Program.cs");
```

Comment:
>.NormalizeLineEndings() [](http://example.com/codeflow?start=19&length=23)

I'm not used to seeing this. Why does this test need it?

Reply:
I'm verifying the actual locations (int Position) of the symbols, so if I don't normalize the line endings I won't get the same test results on windows and linux.

Reply:
Note. Test files should not have their line endings changed.  Line endings have semantic meaning in c#. So you should change whatever tool is doing this to not touch that and treat these test files as pure content that should not be changed.

Reply:
The tool in this case is git

Reply:
A few possible alternatives:
- grab the line/column numbers from the location, which should be the same regardless of the environment's line endings
- don't check the details on the location, just verify the count of `.Locations`

Reply:
Got it. The locations are off by one depending if the line endings are `\n` or `\r\n`.
Current solution seems fine

Reply:
Change your 'git' settings to not change line-endings.  The setting for this is `core.autocrlf=false` :)